From: Stefan Kerkmann <s.kerkmann@pengutronix.de>
Date: Wed, 28 Feb 2024 15:23:49 +0100
Subject: [PATCH] feat(imx8m): add defines for csu_sa access security

This enables the usage of speaking defines instead of magic numbers:

  CSU_SA(CSU_SA_SDMA1, 1, LOCKED)

becomes:

  CSU_SA(CSU_SA_SDMA1, NON_SEC_ACCESS, LOCKED)

Change-Id: Idcabcda677bf7840084a2ea66d321b50aa0b2b20
Upstream-Status: Pending

Signed-off-by: Stefan Kerkmann <s.kerkmann@pengutronix.de>
---
 plat/imx/imx8m/include/imx8m_csu.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/plat/imx/imx8m/include/imx8m_csu.h b/plat/imx/imx8m/include/imx8m_csu.h
index 19d86fd42340..572b0299521a 100644
--- a/plat/imx/imx8m/include/imx8m_csu.h
+++ b/plat/imx/imx8m/include/imx8m_csu.h
@@ -20,6 +20,9 @@
 #define CSU_SEC_LEVEL_6		0x03
 #define CSU_SEC_LEVEL_7		0x0
 
+#define SEC_ACCESS		0x0
+#define NON_SEC_ACCESS		0x1
+
 #define LOCKED			0x1
 #define UNLOCKED		0x0
 
